sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamTest.scala

```diff
@@ -490,6 +490,18 @@ trait StreamTest extends QueryTest with SharedSQLContext with Timeouts {
             try {
               // Add data and get the source where it was added, and the expected offset of the
               // added data.
+              if (currentStream != null &&
```

Comment:
Can you comment here too.  I had no idea what this was doing until read the PR description.